lua/modicator/init.lua

```diff
@@ -24,24 +24,42 @@ local options = {
       ['c']  = M.get_highlight_fg('Constant'),
     },
   },
+  formats = {
+    modes = {
+      ['n']  = { bold = false, italic = false },
+      ['i']  = { bold = false, italic = false },
+      ['v']  = { bold = false, italic = false },
+      ['V']  = { bold = false, italic = false },
+      [''] = { bold = false, italic = false },
+      ['s']  = { bold = false, italic = false },
+      ['S']  = { bold = false, italic = false },
+      ['R']  = { bold = false, italic = false },
+      ['c']  = { bold = false, italic = false },
+    },
+  },
 }
 
 --- Sets the foreground color value of the `CursorLineNr` highlight groups to
 --- `color`.
 --- @param color string
-M.set_highlight = function(color)
+M.set_highlight_and_format = function(color, format)
   local base_highlight = api.nvim_get_hl_by_name('CursorLineNr', true)
-  local opts = vim.tbl_extend('keep', { foreground = color }, base_highlight)
+  -- Should this be defined with these defaults?
+  local base_format = { bold = false, italic = false }
```

Comment:
These make sense as default, but perhaps we should expose these to the user in the configuration API? I have a feeling that most people want the same formatting for all modes :)